src/screens/AppIntroScreen.tsx

```diff
@@ -9,7 +9,7 @@ import { useStaticContent } from '../hooks';
 import { parseIntroSlides } from '../jsonValidation';
 import { AppIntroSlide } from '../types';
 
-const keyExtractor = (slide: AppIntroSlide) => slide.title + slide.text;
+const keyExtractor = (slide: AppIntroSlide, index: number) => `index${index}-text${slide.text}`;
```

Comment:
why have you decided to go with just text and remove title from key?

Reply:
We use it this way in many places in the code and to keep the same structure. I removed it because I don't think we need title if we have id+text.